apps/web/src/app/(default)/dashbord/[slug]/_components/registration-table.tsx

```diff
@@ -69,6 +69,7 @@ export const RegistrationTable = ({
                   .map((r) => r.user.name ?? r.user.email)}
               />
               <DownloadCsvButton slug={slug} />
+              <RemoveAllRegistrationsButton slug={slug} />
```

Comment:
Prøv med å sende `id` istedenfor `slug`. Ref tidligere kommentar.

```suggestion
              <RemoveAllRegistrationsButton id={id} />
```